pkg/auth/asynccache.go

```diff
@@ -10,6 +10,12 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	initializationRetryInterval = 30 * time.Second
```

Comment:
5s

Reply:
What about 10s to match the historical behavior: https://github.com/openshift/console/blob/057e732d0c937215f0d4a78ec6859ccc57d08494/pkg/auth/auth.go#L182